### PR TITLE
Added currency add/subtract to ships cost page

### PR DESCRIPTION
**Pull request by thebwt**
Thing did! it works great. not a 1:1 copy paste, but dang near.

### CODE REVIEW

File: src/templates/actors/ship-sheet.html [hunk 1]
```diff
@@ -426,7 +426,13 @@
                   <input
                     class="w-24 text-center px-1.5 py-1.5 border border-gray-800 bg-gray-400 bg-opacity-75 placeholder-blue-800 placeholder-opacity-75 rounded-md"
                     type="number" min="0" name="data.amountOwed" value="{{actor.data.data.amountOwed}}" />
-                  <label>{{localize 'swnr.sheet.ship.amount-owed'}}</label>
+                  <label>{{localize 'swnr.sheet.ship.amount-owed'}}
+                    <a class="credits-change text-gray-100 hover:text-green-400 transition-colors"
+                    title="{{localize 'swnr.sheet.credit-change'}} {{actor.data.data.tweak.debtDisplay}}"
```
Comment (wintersleepAI) [inline src/templates/actors/ship-sheet.html:431]: The second var will need to change as the tweak attribute is just for actors.

File: src/templates/actors/ship-sheet.html [hunk 2]
```diff
@@ -435,7 +441,13 @@
                   <input
                     class="w-24 text-center px-1.5 py-1.5 border border-gray-800 bg-gray-400 bg-opacity-75 placeholder-blue-800 placeholder-opacity-75 rounded-md"
                     type="number" min="0" name="data.creditPool" value="{{actor.data.data.creditPool}}" />
-                  <label>{{localize 'swnr.sheet.ship.credit-pool'}}</label>
+                  <label>{{localize 'swnr.sheet.ship.credit-pool'}}
+                    <a class="credits-change text-gray-100 hover:text-green-400 transition-colors"
+                    title="{{localize 'swnr.sheet.credit-change'}} {{actor.data.data.tweak.debtDisplay}}"
```
Comment (wintersleepAI) [inline src/templates/actors/ship-sheet.html:446]: same comment

File: src/templates/actors/ship-sheet.html (hunk 1 above)
Comment (thebwt) [reply src/templates/actors/ship-sheet.html:431]: you mean {{actor.data.data.tweak.debtDisplay}} ?

Comment (wintersleepAI) [reply src/templates/actors/ship-sheet.html:431]: yeah that was the field that allowed for custom labels of the 'debt" currency

**Review by wintersleepAI — APPROVED** [review]

**Pull request merged**